daprdocs/content/en/reference/components-reference/supported-pubsub/setup-jetstream.md

```diff
@@ -82,8 +82,32 @@ helm repo add nats https://nats-io.github.io/k8s/helm/charts/
 helm install my-nats nats/nats
 ```
 
-This installs a single NATS server into the `default` namespace. To interact
-with NATS, find the service with: `kubectl get svc my-nats`.
+This installs a single NATS server into the `default` namespace. To interact with NATS, find the service with: `kubectl get svc my-nats`.
+
+Run `docker ps` and verify `my-nats` is running:
+
+```
+CONTAINER ID   IMAGE               COMMAND                  CREATED          STATUS                    PORTS           NAMES
+e0d1b07595d7   9938a89f7aa8        "/prometheus-nats-ex…"   12 minutes ago   Up 12 minutes                  k8s_metrics_my-nats-0_default_32375dda-9b95-4644-b4c8-55518156b0e4_2
+acc025ff5ac1   b08752b7d37f        "nats-server-config-…"   12 minutes ago   Up 12 minutes                  k8s_reloader_my-nats-0_default_32375dda-9b95-4644-b4c8-55518156b0e4_2
+3ff50ac19984   2b71a26d5083        "tail -f /dev/null"      12 minutes ago   Up 12 minutes                  k8s_nats-box_my-nats-box-75bb48bdbf-k4q2f_default_afc76d65-4a32-42b3-bfe1-4ea8a4d4d34c_2
+da03824b1d05   f48fd9739aa3        "nats-server --confi…"   12 minutes ago   Up 12 minutes                  k8s_nats_my-nats-0_default_32375dda-9b95-4644-b4c8-55518156b0e4_2
+```
+
+Open the `nats_my-nats` cluster.
+
+```bash
+docker exec -it da03824b1d052a0dc9270c6d6c3e4ce854726086e3df216c2bf5d3adaaf12b82 /bin/sh
+```
+
+From within the `my-nats` cluster, enable JetStream from the command line:
```

Comment:
This is just a bad recommendation. Instead the comment below should be used -- specifying the command line argument.

The Helm Chart can be modified so that this command line flag can be provided. Instructions for that are here: https://docs.nats.io/running-a-nats-service/nats-kubernetes/helm-charts#jetstream

Someone can either provide all the settings via a `values.yaml` file as is common when using Helm, or folks can inline set the values they need. To enable the Jetstream option for example I think it would be

`helm install my-nats nats/nats --values nats.jetstream.enabled=true`

Reply:
If this recommendation is followed it won't be necessary to `exec` into any container. And doing an `exec` isn't good because anything you change wouldn't persist if the container is restarted.